js/components/portal-dashboard/all-responses-popup/popup-class-nav.tsx

```diff
@@ -14,45 +13,37 @@ import cssClassNav from "../../../../css/portal-dashboard/class-nav.less";
 
 interface IProps {
   anonymous: boolean;
+  handleShowSpotlightDialog: (show: boolean) => void;
```

Comment:
This is probably better named something like `onShowDialog` so it becomes:
`onShowDialog: (show: boolean) => void;`

Reply:
If we use function currying in `StudentResponsePopup`, we can write this as simply:
`onShowDialog: () => void;`